docs/cheaha/tutorial/migrate_conda_envs.md

```diff
@@ -0,0 +1,129 @@
+
+# Migrating Conda Environments using Miniforge
+
+This tutorial will guide you through the steps necessary for migrating Conda environments originally installed with the `default`, `anaconda` or `r` channels to use the Miniforge module and the `conda-forge` or `bioconda` channels.
+
+## Access to Terminal
+
+Access the terminal on Cheaha. You can access the terminal for a compute node on Cheaha, after creating an interactive job. Click on the highlighted part of the image below to access the terminal.
+
+![!Accessing the terminal from a created interactive job on Cheaha](images/comp_node_access.png)
+
+## Steps for Migrating your Conda Environment
+
+1. **Load Anaconda Module**
+You will need to load the Anaconda module using the `module load Anaconda3` command, in a terminal.
+
+1. **Activate the Original Conda Environment**
+Activate the environment you want to migrate:
+
+```bash
+conda activate <your_old_environment>
+
+```
+
+Replace `<your_old_environment>` with the name of your existing Conda environment. In this tutorial the environment `newtest` will be used as an example.
+
+```bash
+conda activate newtest
+
+```
+
+   ![!Steps to export environment](images/anaconda_env_migration.png)
+
+1. **Export the Environment to a .txt File:**
+Export the environment to a `.txt` file, which captures the list of installed packages and their versions:
+
+   ```bash
+   conda list --export > <filename.txt>
+
+   ```
+
+Name the file as you see fit.
+
+   ```bash
+   conda list --export > migrate_env.txt
+
+   ```
+
+When the file has been created, deactivate your Anaconda environment, and do a module reset using the `conda deactivate` and `module reset` commands respectively.
+
+1. **Load the Miniforge Module**
+The next step is to load the Miniforge module on Cheaha using the `module load Miniforge3` command.
+
+1. **Create a New Conda Environment:**
+
+By default Miniforge creates a new environment using the `conda-forge` channel, but for this example we specifically include the option to use the conda-forge and bioconda channels:
+
+   ```bash
+   conda create -n <new_environment_name> --file <environment_backup.txt> -c conda-forge -c bioconda
+
+   ```
+
+Replace `<new_environment_name>` with your desired new environment name. This command will read from the `environment_backup.txt` file and install the packages from the specified channels like `conda-forge` and `bioconda` channels.
+
+   ```bash
+   conda create -n new_env --file migrate_env.txt -c conda-forge -c bioconda
+
+   ```
+
+![!Creating a new environment with Miniforge](images/miniforge_create_env.png)
```

Comment:
I've learned that we don't need the second exclamation point. I'll make an issue about this for the rest of the docs.

That exclamation was for the old gallery view plugin, for some reason. The new one does not need or use it, and it could be annoying for users with screen readers because it appears in the rendered HTML.